### PR TITLE
ref(checkbox): Update focus state

### PR DESCRIPTION
**Pull request by vuluongj20**
Make checkbox focus states more similar to those on our buttons.

**Before ——**
Screenshot 2023-02-10 at 10 45 16 AM
Screenshot 2023-02-10 at 10 44 58 AM

**After ——**
Screenshot 2023-02-10 at 10 44 28 AM
Screenshot 2023-02-10 at 10 44 44 AM

### CODE REVIEW

File: docs-ui/stories/components/checkbox.stories.js [hunk 1]
```diff
@@ -19,52 +20,36 @@ export default {
...
   return (
     <div>
-      <Checkbox
-        {...props}
-        checked={checked}
-        onChange={e => setChecked(e.target.checked)}
-      />
+      <Checkbox {...props} onChange={e => updateArgs({checked: !!e.target.checked})} />
     </div>
   );
```
Comment (vuluongj20) [inline docs-ui/stories/components/checkbox.stories.js:36]: Make use of storybook's `useArgs` hook to automatically update the `checked` arg in the control panel whenever we click on the checkbox:
https://user-images.githubusercontent.com/44172267/218173071-443e63e4-029f-45eb-a6af-6e0e37030afe.mov

File: docs-ui/stories/components/checkbox.stories.js [hunk 2]
```diff
@@ -19,52 +20,36 @@ export default {
...
   return (
-    <div>
-      <Label>
-        Label to left
-        <Checkbox
-          {...props}
-          checked={check1}
-          onChange={e => setCheck1(e.target.checked)}
-        />
-      </Label>
```
Comment (vuluongj20) [inline docs-ui/stories/components/checkbox.stories.js:52]: I don't think we want to encourage putting labels on the left side of checkboxes.

**Review by evanpurkhiser — APPROVED** [review]
yes!!!!

File: docs-ui/stories/components/checkbox.stories.js (hunk 1 above)
Comment (malwilley) [reply docs-ui/stories/components/checkbox.stories.js]: Ah, good to know about `useArgs`!

Comment (vuluongj20) [reply docs-ui/stories/components/checkbox.stories.js]: I just learned about it today. It's great!

**Pull request merged**